### PR TITLE
Allow to test debian packages

### PR DESCRIPTION
**Pull request by sergio-costas**
Sometimes it is desirable to test .deb packages in the core22 system before uploading them to the core-desktop PPA.

This PR allows to do this, by copying any .deb file placed in the "test-debs" folder in the project dir, and making them available to APT with the maximum priority, thus overriding any other package with the same name.

### CODE REVIEW

File: build-package.sh [hunk 1]
```diff
@@ -28,6 +28,9 @@ case "$1" in
         ;;
     stage)
         craftctl default
+        if [ -e ${CRAFT_PROJECT_DIR}/test-debs ]; then
```
Comment (robert-ancell) [inline build-package.sh:31]: Might be a good idea to put a comment above this.

File: build-package.sh [hunk 2]
```diff
@@ -28,6 +28,9 @@ case "$1" in
...
+            /bin/cp -a ${CRAFT_PROJECT_DIR}/test-debs/* ${CRAFT_STAGE}/local-debs/
```
Comment (robert-ancell) [inline build-package.sh:32]: Why the full path to cp?

File: build-package.sh (hunk 1 above)
Comment (sergio-costas) [reply build-package.sh:37]: You are right. Done.

File: build-package.sh (hunk 2 above)
Comment (sergio-costas) [reply build-package.sh]: Ok, my fault... that error was in another place. Fixed.

**Review by kenvandine — APPROVED** [review]
This will be very useful, looks good.

**Pull request merged**